### PR TITLE
Mention `no merge policy` in the CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by stepnivlk**
Issue: https://github.com/rust-lang/rust/issues/59233

### CODE REVIEW

**Review by oli-obk — COMMENTED** [review]
not sure who to ping on this policy, I'll drop some hints in zulip and discord

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -119,6 +119,10 @@ bring those changes into the source repository.
 
 Please make pull requests against the `master` branch.
 
+Rust follows no merge policy, meaning you are expected to always rebase instead of merge.
+E.g. always use rebase when brining latests changes from master to your feature branch.
```
[the remark this answers is not in the document]
Comment (davidtwco) [reply CONTRIBUTING.md]: In addition to @oli-obk's suggestion above:

```suggestion
E.g. always use rebase when brining latest changes from master to your feature branch.
```

**Comment by stepnivlk** [comment]
@oli-obk @davidtwco thx for reviews! time to find some english course.

**Comment by Centril** [comment]
@oli-obk Ping from triage... are we happy?

**Comment by oli-obk** [comment]
@bors r+ rollup

jup, thanks!

**Pull request merged**